bpf/init.sh

```diff
@@ -627,10 +589,14 @@ for iface in $(ip -o -a l | awk '{print $2}' | cut -d: -f1 | cut -d@ -f1 | grep
 	done
 	$found && continue
     for where in ingress egress; do
-        if tc filter show dev "$iface" "$where" | grep -q "bpf_netdev.o"; then
+        if tc filter show dev "$iface" "$where" | grep -qP "bpf_netdev.o"; then
```

Comment:
nit: It'd be good to add a comment here stating that this old prog name was loaded onto devices in Cilium 1.7 or earlier and this snippet could be removed in v1.9.